app/models/solidus_afterpay/order_component_builder.rb

```diff
@@ -98,8 +98,28 @@ def item(line_item)
         price: ::Afterpay::Components::Money.new(
           amount: line_item.price.to_s,
           currency: line_item.currency
-        )
+        ),
+        preorder: check_date(line_item),
+        estimated_shipment_date: estimated_shipment_date(line_item)
       )
     end
+
+    def estimated_shipment_date_product(line_item)
+      line_item.product.property("estimatedShipmentDate")
+    end
+
+    def estimated_shipment_date_variant(line_item)
+      line_item.variant.variant_properties.reverse.find{ |prop| prop.property.name == "estimatedShipmentDate" }&.value
```

Comment:
why did you need to call `reverse` on the variant_properties?

Reply:
this was related to rubocop

Reply:
Removing the `reverse` method doesn't trigger any rubocop offence 🤔

Reply:
I needed the last, not first